alkos/kernel/include/todo.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef KERNEL_INCLUDE_TODO_HPP_
```

Comment:
I don't intuitively get what's the use case. Maybe explain this a bit using a real world scenario plz

Reply:
This is long discussion. Will be addressed during some sync call.